SPECS/linuxptp/linuxptp.spec

```diff
@@ -24,13 +24,24 @@ Source10:	linuxptp-testsuite-%{testsuite_ver}.tar.gz
 Source11:	clknetsim-%{clknetsim_ver}.tar.gz
 
 # fix handling of zero-length messages
-Patch0:		linuxptp-zerolength.patch
+# Patch0:		linuxptp-zerolength.patch
 # revert phc2sys options needed by the older version of test suite
-Patch1:		clknetsim-phc2sys.patch
+Patch0:		clknetsim-phc2sys.patch
 
 # The following patch is a combination of multiple patches to enable HA in linuxptp
 # https://review.opendev.org/c/starlingx/integ/+/891638
-Patch2:         enable-ha.patch
+# Patch2:         enable-ha.patch
```

Comment:
why is this being removed?

Reply:
there are 47 patches in this one single patch, and a few patches are already part of linuxptp v4.3 while remaining are enable HA. These HA patches are no found in upstream project linuxptp and its not part of mailing list. Hence this patch is applicable for linuxptp v3.1.1 version of AZL contributed by Red Hat only. TSN enabling doesn't require this HA patch